File: src/Python/easy/string/problem1768.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        maxLength = max([len(word1), len(word2)])
        res = ''
        for i in range(maxLength):
            res += word1[i] if i < len(word1) else '' 
            res += word2[i] if i < len(word2) else '' 
        return res


word1 = "ab";  word2 = "pqrs"


solution = Solution()
solution.mergeAlternately(word1, word2)

File: src/JavaScript/easy/string/problem1768.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    let maxLength = word1.length > word2.length ? word1.length : word2.length
    let res = ''
    for(let i = 0; i < maxLength; i++){
        res += word1.split("")[i] ? word1.split("")[i] : ''
        res += word2.split("")[i] ? word2.split("")[i] : ''
    }
    return res
};

// let word1 = "abc", word2 = "pqr"
let word1 = "ab", word2 = "pqrs"


mergeAlternately(word1, word2)